Polymorphism/Hierarchy/Program.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Hierarchy
+{
+    class Program
+    {
+        public static string GetString(string messageToUser)
+        {
+            Console.WriteLine(messageToUser);
+            string userInput = "";
+            userInput = Console.ReadLine();
+            Console.WriteLine($"string read ({userInput})");
+            return userInput;
+        }
+        public static decimal GetDecimal(string messageToUser)
+        {
+            Console.WriteLine(messageToUser);
+            decimal userInput = -1;
+            decimal.TryParse(Console.ReadLine(), out userInput);
+            if (userInput == 1) 
+            {
+                Console.WriteLine("Failed to read try again");
+                GetDecimal(messageToUser);
+            }
+            Console.WriteLine($"number read ({userInput})");
+            return userInput;
+        }
+        public static Animal SpawnAnimal()
+        {
+            string animalType = "";
+            bool loop = true;
+            Animal someAnimal = null;
+            while (loop)
+            {
+                animalType = GetString("Enter one of the folowing:\n" +
+                "Mouse\n" +
+                "Cat\n" +
+                "Zebra\n" +
+                "Tiger\n");
+
+                if (animalType.Equals("Mouse"))
+                {
+                    loop = false;
+                    someAnimal = (new Mouse(
+                        GetString("Enter the name of this creature:\n"),
+                        (double)GetDecimal("Enter animal weight:\n"),
+                        (int)GetDecimal("Enter food needed to feed this creature:\n"),
+                        GetString("Enter living envireoment of this creature:\n")
+                        ));
+                }
+                if (animalType.Equals("Zebra"))
+                {
+                    loop = false;
+                    someAnimal = (new Zebra(
+                        GetString("Enter the name of this creature:\n"),
```

Comment:
Šis tev atkārtojas jau iepriekš. Varēji iznest atsevišķā metodē un pēc tam izsaukt to metodi